### DISCUSSION TITLE
commercial version of ihatemoney

### DISCUSSION THREAD

**Discussion by Darin755** [spiral-project/ihatemoney#1089] [Ideas]
Hello,
Is there a commercial version of this software? I know many companies would love to ditch QuickBooks for something better. It would be nice to self host this and then pay for a off site service that does payrole based on the on site database. 

Having a commercial version would be good because it would provide funding that could allow for more features.

**Comment by Glandos** [comment]
Hi!

Yes, a commercial version would be a booster for this project maintenance, but unfortunately, the current team is not focused on that. However, if some people do want to do that, please contact us :)

**Comment by almet** [comment]
Hi, it seems that ihatemoney doesn't share the same goals as QuickBooks. That being said, I second what @Glandos is saying.
